src/App.vue

```diff
@@ -603,13 +604,26 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     await updateLdUser(key, email, firstName, lastName, custom)
   }
 
+  /** Perform high level component validations before proceeding to summary. */
+  private async validateComponents (): Promise<void> {
+    // Verify component validations
+    await this.setValidResolutionDate(this.getIsResolutionDatesValid)
+
+    // evaluate valid flags. Scroll to invalid components or continue to review.
+    if (this.validateAndScroll(this.getValidComponentFlags, ComponentFlags)) {
+      // We don't change views just interchange components, so scroll to top for better UX.
+      this.scrollToTop(document.getElementById('app'))
+      this.setSummaryMode(true)
+    }
+  }
+
   /** Perform high level validations before filing. */
   private async validateApp (): Promise<void> {
     // Prompt app validations
     this.setAppValidate(true)
 
     // evaluate valid flags. Scroll to invalid components or file alteration.
-    if (this.validateAndScroll(this.getAlterationValidFlags)) await this.onClickSave(false)
+    if (this.validateAndScroll(this.getAlterationValidFlags, ReviewSummaryFlags)) await this.onClickSave(false)
```

Comment:
Just a small nit: when the code is more than trivial, I prefer to see the usual braces style.
```
if (condition) {
  do something
}
```

Reply:
No prob at all!
sounds good, will update

Reply:
Updated!